tools/export/__init__.py

```diff
@@ -93,12 +96,13 @@ def export(project_path, project_name, ide, target, destination='/tmp/',
                     use_progen = True
             except AttributeError:
                 pass
+
+            if target not in Exporter.TARGETS or ide.upper() not in TARGET_MAP[target].supported_toolchains:
+                supported = False
```

Comment:
I think it's useful to have those targets as a list (scripts used it to enable exporting in the online IDE)  ,but not hard coded (progen provides API to check if target is supported), the way we do for uvision. Was that your question?